src/Compilers/VisualBasic/Portable/Operations/VisualBasicOperationFactory.vb

```diff
@@ -256,12 +255,11 @@ Namespace Microsoft.CodeAnalysis.Semantics
 
         Private Function CreateBoundAssignmentOperatorOperation(boundAssignmentOperator As BoundAssignmentOperator) As IOperation
             Dim kind = GetAssignmentKind(boundAssignmentOperator)
-            Dim isImplicit As Boolean = boundAssignmentOperator.WasCompilerGenerated
             If kind = OperationKind.CompoundAssignmentExpression Then
```

Comment:
[BoundAssignmentOperatior](http://source.roslyn.io/#Microsoft.CodeAnalysis.VisualBasic/Generated/BoundNodes.xml.Generated.vb,b1659918c50dce2c,references) in VB doesn't contain a Checked flag, so I have defaulted it to false. I will file an issue to track the correct way for calculated IsLifted.